api/integrations/fifabot/commands/addMatch.js

```diff
@@ -0,0 +1,57 @@
+const request = require('request-promise');
+
+const pattern = /^add match (.*) (.*) (\d+)\s*x\s*(\d+) (.*) (.*)$/;
+const seasonOptions = {
+  url: 'http://localhost:5001/api/championships',
```

Comment:
This will be changed later on to use the environment variables, right? Localhost isn't going to be the correct one.

Also, can't you use the baseAPI url and just concatenate with the specific endpoint later on?